### PR TITLE
Feat: Create post (blog) layout and styles

### PR DESCRIPTION
**Pull request by llandkoer**
Add post (blog) layout [US32](https://trello.com/c/rWLGzoXE/40-us32)

### CODE REVIEW

File: src/layout/post.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        <h1 class="post__title">uenfuwef ewfeeuwfb</h1>
+        <div class="post__text">
+          <p class="post__author">Autor: XXXXX</p>
+          <p class="post__date">Fecha: XXXX</p>
+        </div>
+      </div>
+      <div class="post__information">
+        <p>urfuefuervevuefbeufbefuerfuberfu</p>
+        <p>
+          jnerfunreuenvuenvubvyerbuernurv rvuerbfur fuer fuer ferfuebferuf eruf eruf ryu feruf erhf eruf erfh erf eryf eryfer yfer fyer ffyerbfyef
```
Comment (NataliaOrtizMa) [inline src/layout/post.html:24]: I suggest to use a _Lorem Ipsum_ text here

**Review by NataliaOrtizMa — COMMENTED** [review]
I suggest to use a _Lorem Ipsum_ text in the `div class='post__information` of the post.html file

**Comment by llandkoer** [comment]
Thanks for the suggestion. I will implement this change as soon as possible.

**Review by NataliaOrtizMa — APPROVED** [review]
Well done!

**Review by Jeferguto — APPROVED** [review]
Everything is fine, good job!

**Pull request merged**